src/containers/Technician/AddTechinician/index.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { Button, Col, DatePicker, Form, Input, Row, Select, Switch } from 'antd';
+import { PlusOutlined } from '@ant-design/icons';
+
+import styles from './style.module.css';
+import Modal from '../../../components/Modal';
+import { validatePlate } from '../../../utils/validators';
+
+const ModalNewCar = ({ closeModal, form, saveModal, title, visible }) => {
+  return (
+    <Modal
+      closeModal={closeModal}
+      saveModal={() => form.submit()}
+      title={title}
+      visible={visible}
+    >
+      <Form form={form} name="newCar" layout="vertical" onFinish={saveModal}>
+        <Form.Item name="model" label="Modelo" rules={[{ required: true }]}>
+          <Input placeholder="Digite o modelo" />
+        </Form.Item>
+
+        <Row gutter={24}>
+          <Col span={12}>
+            <Form.Item name="year" label="Ano" rules={[{ required: true }]}>
+              <DatePicker
+                className={styles.datePicker}
+                picker="year"
+                placeholder="selecione o ano"
```

Comment:
```suggestion
                placeholder="Selecione o ano"
```
Corrigir ortografia, SEMPRE COMEÇAR COM LETRAS MAIÚSCULAS!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Reply:
a quantidade de ponto de exclamação em uma exclamação não a intensifica. ;)